Index.json

```diff
@@ -0,0 +1,59 @@
+[

```

Comment:
The index has been re-named as Index.json and is currently (and should remain) in alphabetical order.

Reply:
Interested to understand why it must be alphabetical? Seems like that could get out of sync quickly and could surely be resolved by ordering when you retrieve the file etc.?

Reply:
Helps maintainability

Reply:
RB had suggested it on a previous revision, has no purpose to the code but it helps readability for quick additions, or to find missing items as the folders are sorted alphabetically as well.